obspy/io/nordic/core.py

```diff
@@ -466,6 +474,50 @@ def _read_uncertainty(tagged_lines, event):
     return event
 
 
+def _read_highaccuracy(tagged_lines, event):
+    """
+    Read high accuracy origin line.
+    """
+    if 'H' not in tagged_lines.keys():
+        return event
+    # In principle there shouldn't be more than one high precision line
```

Comment:
Is this true?

Reply:
Actually this is not written in the SEISAN file, although I don't know how you would associate multiple 'H' lines with one (or multiple) "1" lines.  I was just being lazy I guess.